src/DependencyInjection/Compiler/AdminAddInitializeCallCompilerPass.php

```diff
@@ -26,9 +27,9 @@ final class AdminAddInitializeCallCompilerPass implements CompilerPassInterface
 {
     public function process(ContainerBuilder $container): void
     {
-        $admins = $container->findTaggedServiceIds('sonata.admin');
+        $admins = $container->findTaggedServiceIds(TaggedAdminInterface::ADMIN_TAG);
         foreach (array_keys($admins) as $id) {
             $container->getDefinition($id)->addMethodCall('initialize');
         }
     }
-}
+}
```

Comment:
Please configure your IDE to not remove the extra EOF line

Reply:
@VincentLanglet my bad ! should be fixed